docs/register-create-stake-phase-2.md

```diff
@@ -0,0 +1,227 @@
+unbonding transaction can be found [here](./staking-script.md#unbonding-transaction).
+
+`slashing_tx`: 
+The slashing transaction is in hex format. Generated during both staking and 
+unbonding phases, this transaction spends the staking or unbonding transaction 
+in cases of protocol violations such as double-signing. To see exactly what 
+constitutes a valid slashing transaction, see the 
+[`Slashing Transaction` section](./staking-script.md#slashing-transaction).
+A valid signature for this transaction must be included in the 
+`slashing_tx_sig` field.
+
+`delegator_slashing_sig`: 
+The signature for the `slashing_tx`. This must be a valid signature 
+generated using the private key corresponding to the BTC public key (`btc_pk`).
+
+`delegator_unbonding_slashing_sig`: 
+The unbonding slashing signature for the delegator. This must be a valid 
+signature generated using the private key corresponding to the BTC public key 
+(`btc_pk`).
+
+`pop` (Proof of Possession):
+A cryptographic signature verifying ownership of the Bitcoin private key used 
+for staking. Required for registering a phase-1 stake to confirm the legitimate 
+ownership of the associated Bitcoin address. This will be generated during the 
+phase-1 creation of stake process. To see the full message, see the 
+[`ProofOfPossessionBTC` message](./proto/babylon/btcstaking/v1/pop.proto).
+
+`inclusion_proof` (Optional): 
+A Merkle proof showing that the staking transaction is included in the 
+Bitcoin chain. This field is optional but recommended. To see the full message, 
+see the [`InclusionProof` message](./proto/babylon/btcstaking/v1/btcstaking.proto).
+
+If not provided during submission, it must be submitted later using the 
+`MsgAddBTCDelegationInclusionProof` message. To see the full message, see the 
+[`MsgAddBTCDelegationInclusionProof` message](./proto/babylon/btcstaking/v1/tx.proto).
+
+`Other General Fields`:
+- `staking_value`: The amount of BTC locked for delegation (in satoshis).
+- `staking_time`: The timelock period for the staking transaction (in blocks).
+- `unbonding_time`: The timelock period for unbonding (in blocks).
+
+## 3. Stake creation and registration
+### 3.1. Registering a phase-1 stake
+
+To register a Bitcoin stake that is already on Bitcoin to 
+the Babylon chain (e.g., a phase-1 stake), you need to submit a 
+`MsgCreateBTCDelegation` with a valid inclusion proof. This requires your 
+staking transaction to already be confirmed on the Bitcoin network. You can do 
+this either manually through the Bitcoin network or using the staker library.
+
+1. Create `MsgCreateBTCDelegation` with the inclusion proof filled as defined 
+above and submit `MsgCreateBTCDelegation` to the Babylon network's btcstaking 
+module. You will then wait for the covenant committee to submit their signatures.
+
+
+Alternatively, you can use the CLI to register phase-1 stake:
```

Comment:
No need to specify CLI instructions here. It can be one of the options to construct and communicate the message.